docs/schedule/README.md

```diff
@@ -54,14 +54,28 @@ You can create rooms by clicking on Admin on the top right, then Rooms.
 ### Options
 ![Reservation options](https://attachments.cbd.int/apps/eunomia/reservation-options.png)
 
-- In the Options tab, you can specify certain options specific to your event such as public accesseibility, confirmation status and security presence at the venue.
-- Similar to Details, information entered in the Options window will be shown on the Reservations page.
+- In the Options tab, you can specify certain options specific to your event such as public accessibility, confirmation status and security presence at the venue.
 
 ### CCTV
 ![Reservation cctv animated gif](https://attachments.cbd.int/apps/eunomia/reservation-cctv.gif)
 
-- Click on the CCTV tab to enter any text you would like to show on the CCTV screen
-- When you are done entering the event's details, click on Save to finalize the reservation
+- Click on the CCTV tab to enter any text you would like to be shown on the CCTV screen
+- If applicable, choose the level of significance of your message. 
+
+::: tip
+The first level is a **Warning**, which could be a simple notification and the second level is an **Alert**, which implies something more substantial such as a cancellation.
+:::
+
+![Warn cctv animated gif](https://attachments.cbd.int/apps/eunomia/reservation-warn-cctv.png)
+
+![Alert cctv animated gif](https://attachments.cbd.int/apps/eunomia/reservation-alert-cctv.png)
+
+![cctv main schedule gif](https://attachments.cbd.int/apps/eunomia/cctv-main-schedule.png)
+
+::: tip
+This is the CCTV view. You can click on the top bar to change to the next screen load.
```

Comment:
I think this may be better placed on this page once it starts to come together.  http://scbd.github.io/eunomia.cbd.int/cctv/  The cctv tab on the reservation is just to post a msg with respect to the reservation appearing on the cctv.